### PR TITLE
Add news banner

### PR DESCRIPTION
**Pull request by Bobfrat**
@DanielJMaher @FBRTMaka @birdage 
At 11 am the video turns on again, so I'll test that the banner appears at the correct times. Hold on until then to merge.

### CODE REVIEW

**Comment by birdage** [comment]
:+1:

**Comment by Bobfrat** [comment]
Seemed to work, the banner came up at 1pm UTC (11 am EST) and when I refreshed the page at 15 past the hour the banner was removed.

**Comment by FBRTMaka** [comment]
@Bobfrat yeah looked sweet!

File: ooiui/static/css/custom/banner.css [hunk 1]
```diff
@@ -136,5 +136,219 @@ div#wrapper-asset {
...
+  -moz-animation-iteration-count: 1;
+  -webkit-animation-delay: 0.5s; 
+  animation-delay: 0.5s;
+}
+
+.fadein {
+  -webkit-animation-name: fadein;
+  -moz-animation-name: fadein;
+  -o-animation-name: fadein;
+  animation-name: fadein;
```
Comment (FBRTMaka) [inline ooiui/static/css/custom/banner.css]: @Bobfrat , was this supposed to have a keyframe?

File: ooiui/static/css/custom/banner.css [hunk 2]
```diff
@@ -136,5 +136,225 @@ div#wrapper-asset {
...
+  position: absolute;
+  font-family: arial;
+  font-size: 13px;
+  margin-top: -22px;
+  color: white;
+  margin-left: 125%;
+  padding-top: 3px;
+  overflow : hidden;
+  white-space: nowrap;
+}
```
Comment (Bobfrat) [inline ooiui/static/css/custom/banner.css]: @FBRTMaka I added ```overflow: hidden``` and ```white-space: nowrap``` to fix the scrolling issue.

Comment (FBRTMaka) [inline ooiui/static/css/custom/banner.css]: @Bobfrat sweet!

**Pull request merged**